roles/mongodb/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Install mongodb on {{ ansible_distribution }}
-  include: mongodb_{{ ansible_pkg_mgr }}.yml
+  import_tasks: mongodb_{{ ansible_pkg_mgr }}.yml
```

Comment:
Using a var in the file name == needs to be `include_tasks`

Reply:
Done